bin/foreman-proxy-certs-generate

```diff
@@ -28,5 +28,32 @@ Kafo::KafoConfigure.hooking.register_pre(:init) do
   org_param.value = organization
 end
 
+Kafo::KafoConfigure.hooking.register_pre_commit(:certs_check) do
+  ca_file   = param('certs', 'server_ca_cert').value
+  cert_file = param('certs', 'server_cert').value
+  key_file  = param('certs', 'server_key').value
+
+  unless app_value(:certs_skip_check) || [ca_file, cert_file, key_file].all?(&:nil?)
+    command = ['katello-certs-check', '-t', 'foreman-proxy', '-c', cert_file, '-k', key_file, '-b', ca_file]
```

Comment:
This uses the unbranded version. Now I'm debating if we should even pass it at all. Right now it's only used to output commands, which are discarded if the command passes. Should we drop `-t foreman-proxy`?

Reply:
Pass what?

Reply:
The `-t foreman-proxy` part. For Satellite it should be `-t capsule`, but looking at the code this is the only place it's used:
https://github.com/theforeman/foreman-installer/blob/8b7b58dfd4b5d50a391b6c21d9b8db777896fc7f/bin/katello-certs-check#L251C7-L251C7

It's only printed if the exit code is 0, but then we discard the output. So should it be
```suggestion
    command = ['katello-certs-check', '-c', cert_file, '-k', key_file, '-b', ca_file]
```

Or do we imagine the target to become something we care about in our checks?

Reply:
Yes - let's drop it.